bootcamp-java/Topic1/classes/Offers.java

```diff
@@ -0,0 +1,76 @@
+package classes;
+
+import java.util.ArrayList;
+import classes.Products;
+
+public class Offers {
+	
+	private String description;
+	private float discount;
+	private Products product;

```

Comment:
me quedó creado y me había olvidado de borrarlo.
ya está actualizado.

Date: Wed, 21 Jan 2015 11:50:42 -0800
From: notifications@github.com
To: java-bootcamp@noreply.github.com
CC: marcosadriandalmasso@hotmail.com
Subject: Re: [java-bootcamp] Modificaciones solicitadas (#6)

In bootcamp-java/Topic1/classes/Offers.java:

> @@ -0,0 +1,76 @@
> +package classes;
> +
> +import java.util.ArrayList;
> +import classes.Products;
> +
> +public class Offers {
> +	
> +	private String description;
> +	private float discount;
> +	private Products product;

Que es este producto individual?


—
Reply to this email directly or view it on GitHub.

  
    
    
  
  
 		 	   		  =